cmd/server/assets/login.html

```diff
@@ -0,0 +1,107 @@
+{{define "login"}}
+<!doctype html>
+<html lang="en">
+
+<head>
+    {{template "head" .}}
+
+    {{template "firebase" .}}
+</head>
+
+<body>
+    {{template "navbar" .}}
+
+    <main role="main" class="container">
+        {{template "flash" .}}
+        <div class="alert alert-warning" role="alert" id="message"></div>
+
+        <div class="card mb-3">
+            <div class="card-body">
+                <form>
+                    {{ .csrfField }}
+                    <div class="row form-group">
+                        <label for="email" class="col-sm-3">Email</label>
+                        <div class="col-sm-9">
+                            <div class="input-group p-0 shadow-sm">
+                                <input type="text" id="email" name="email" class="form-control" />
```

Comment:
Nit: `type=email` is more accurate and will show the right keyboard on mobile

Reply:
Thanks! I'll incorporate into my next